local-volume/README.md

```diff
@@ -165,30 +165,68 @@ $ kubectl create -f bootstrapper/deployment/kubernetes/example-storageclass.yaml
 
 ### Step 3: Creating local persistent volumes
 
-#### Option 1: Bootstrapping the external static provisioner
+#### Option 1: Generating Provisioner's daemonset spec and config map by using Helm chart 
 
-This is optional, only for automated creation and cleanup of local volumes. See
-[bootstrapper/](./bootstrapper) and [provisioner/](./provisioner) for details and
-sample configuration files.
+This option uses helm templating to generate the provisioner's DaemonSet and ConfigMap specs.
+The generated specs can be further customized as needed (usually not necessary), and then deployed using kubectl.
 
-1. Create an admin account with cluster admin priviledge:
+**helm template** uses 3 sources of information:
+1. Provisioner's chart template located at helm/provisioner/templates/provisioner.yaml
+2. Provisioner's default values.yaml which contains variables used for rendering a template.
+3. (Optional) User's customized values.yaml as a part of helm template command. User's provided
+   values will override default values of Provisioner's values.yaml.
+
+Default values.yaml is located in local-volume/helm/provisioner folder, user should not remove variables from this file but can
+change any values of these variables. 
+The only exception is the list of storage classes and respective mounting points. In general there should be at least one storage class
```

Comment:
Maybe instead of calling out specific variables here, it would be better to document them as comments in values.yaml

Reply:
done